notes/themes/proposed.th.txt

```diff
@@ -506,5 +506,57 @@ human mental condition
 https://en.wikipedia.org/wiki/Delusional_disorder
 
 
+speculative robot
+=================
+
+:: Description
+There is a mobile machine equipped with an advanced artificial intelligence.
+For example, an android or a robotic dog that looks and behaves much like the
+real article.
+
+:: Parents
+artificial intelligence
+
+:: Notes
+For this theme to apply, the machinery itself or the artificial intelligence
+must be well beyond what is possible when the story was first released. The
+theme does not apply to creature-like machines that lack relatively advanced
+AI. For example, the theme does not apply to animal-shaped vehicles like the
+iconic AT-AT walker from Star Wars or the mechas from Japanese anime, even
+though these things are sometimes referred to as robots. The "piloted
+creature-shaped vehicle" theme is used to cover such contraptions.
+
+:: References
+https://en.wikipedia.org/wiki/Robots_in_literature
+
+
+piloted creature-shaped vehicle
+===============================
+
+:: Description
+There is a belimbed (or betentacled etc.) mobile automaton that is piloted by
```

Comment:
I don't think "automaton" is the right word here.

Reply:
Personally I think a tentacle is a kind of limb, but I guess we should consult a marine biologist and a linguist about that... 🙃

Reply:
I guess something like a golem might be considered to be an automaton. What would you use in place of automaton? mechanical contraption?